data/sql/updates/pending_db_world/rev_1640263955874251100.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1640263955874251100');
+
+-- Set 100% drop chance for Abyssal crest in Crimson Templar / Azure Templar / Hoary Templar / Earthen Templar
+UPDATE `creature_loot_template` SET `Chance`='100' WHERE `Item`=20513 AND `Reference`=0 AND `GroupId`=0;
```

Comment:
> AND `Reference`=0 AND `GroupId`=0;

Not needed I know I know, but sometimes I don't trust myself

Reply:
Hahaha, det är okej